package.json

```diff
@@ -72,12 +72,16 @@
     "serverless-offline": "^3.18.0",
     "serverless-webpack": "^4.3.0",
     "synp": "^1.2.1",
+    "stylelint": "^8.4.0",
+    "stylelint-config-standard": "^17.0.0",
+    "stylelint-no-unsupported-browser-features": "^1.0.0",
```

Comment:
Are these required for `@hollowverse/common`? Why?

Reply:
`serverless-webpack` was bundling these as dependencies of `@hollowverse/common` (even though they are just peer dependencies). Explicitly listing these as development dependencies forced `serverless-webpack` to exclude them in the production bundle.

Reply:
Hmmm, that's not how I thought `peerDependencies` worked. I thought they had to be explicitly listed in the downstream package for them to be downloaded. 

Interesting...

Reply:
You are right. It's not how `peerDependencies` work. But serverless is doing its own things with the dependencies because Lambda does not install any packages. They have to be bundled with the ZIP file that gets uploaded as the function code, and it looks like serverless/`serverless-webpack` is not dealing correctly with `peerDependencies`.